lib/services/dialog_import_service.rb

```diff
@@ -122,6 +126,8 @@ def import_from_dialogs(dialogs)
         )
       )
       fields = new_or_existing_dialog.dialog_fields
+      associations = build_association_list(dialog)
+      next if associations.empty?
```

Comment:
I don't think this is necessary, right? There's nothing after the `.each` block that we're trying to guard against running, and the `.each` won't do anything if the `associations` list is empty anyway.